transactions/set-tx-fee-parameters/2025/dec-1/arguments.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
```

Comment:
The [FLIP proposes](https://github.com/onflow/flips/pull/352/files#diff-b262cdb452bb8a6cb50a3b5503dcfca05e9befba668b907f0080230669cf3b02R56-R59):

1. Unit Cost of Execution Effort be set to 4E-05 .
2. Inclusion Fee be set to 1E-4 FLOW .

Since the execution effort should be specifed as `value * 1e8`, setting to to 4000.